website/docs/documentation/Pipelines/Optimus_Pipeline/README.md

```diff
@@ -235,3 +254,17 @@ The Optimus pipeline is a single sample pipeline, but it can accept multiple FAS
 
 Parameters are listed in each task WDL. For a list of the tasks, see the table in the [Task Summary Section](#optimus-task-summary). Select the link for the task of interest and then view the parameters in the task WDL "command {}" section. For the task Docker image, see task WDL "# runtime values" section; the Docker is listed as "String docker =  ". If you want to learn more about all the different parameters available for a software tool, please select the relevant link in the table's "Tool" column.
 :::
+
+::: details How does Optimus compare to Cell Ranger? 
+
+Cell Ranger is a commonly used set of analysis pipelines developed by [10x Genomics](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger). Optimus and Cell Ranger share many features and additionally, Optimus results are validated against Cell Ranger results (see our [human validation report](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/optimus/benchmarking/v1_Apr2019/optimus_report.rst)). 
+
+*So why develop an independent pipeline for 10x data analyses?* 
+
+For three reasons:
+1) Need for an open source, cloud-optimized pipeline. When Optimus was developed, Cell Ranger software was not yet open source, nor was it optimized for the cloud. To date, the Cell Ranger open source code is still not regularly updated with Cell Ranger releases. In consequence, using the latest Cell Ranger (which is not open source yet) limits our ability to harness the breadth of tools available in the scientific community. 
```

Comment:
on that last sentence, it's that and it's just not transparent.. not sure if that's as important to others as it is to me lol.

Reply:
@kbergin When you say "it's not transparent"- are you referring to Cell Ranger not being transparent (or the sentence)?